### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by iraj-jelo**
a `requirements.txt `and some link fixes in `bot.py` added to the project

### CODE REVIEW

File: main.conf [hunk 1]
```diff
@@ -8,7 +8,7 @@ plugin_dir = .
 plugins = test,test2
 sasl_password = badtestpassword
 sasl_username = alefir-bot
-
+deop_owner = False
```
Comment (bigfoot547) [inline main.conf:11]: Don't forget about usermode 😛

**Review by bigfoot547 — CHANGES_REQUESTED** [review]

File: main.conf (hunk 1 above)
Comment (iraj-jelo) [reply main.conf:11]: @bigfoot547  done 👇

**Review by bigfoot547 — APPROVED** [review]

**Comment by bigfoot547** [comment]
Sorry, I can't merge now, I need git

**Comment by bigfoot547** [comment]
32516ad

**Pull request closed without merging**